Makefile

```diff
@@ -307,33 +307,26 @@ ARCH ?= amd64
 IMAGE_TAG ?= $(shell git describe --tags)
 GO_LDFLAGS='$(shell hack/make-rules/version.sh)'
```

Comment:
Seems all these value can be removed?

Reply:
done, and now both helm_e2e and `make image` cmd use default image tag defined by cmd `shell git describe --tags`